sdk/customers.go

```diff
@@ -11,3 +14,19 @@ func ValidateCustomers() bool {
 	}
 	return true
 }
+
+//ValidateCustomerList validate customer names list
```

Comment:
where are we calling this function?

Reply:
This will be called inside function once we release new version of sdk and the vendor it inside the function.